pkgs/os-specific/linux/kernel/manual-config.nix

```
@@ -427,77 +426,80 @@ lib.makeOverridable (
         # which trips the broken symlink check. So, just skip it. We'll know if it explodes.
         dontCheckForBrokenSymlinks = true;
 
-        postInstall = optionalString isModular ''
-          mkdir -p $dev
-          cp vmlinux $dev/
-          if [ -z "''${dontStrip-}" ]; then
-            installFlagsArray+=("INSTALL_MOD_STRIP=1")
-          fi
-          make modules_install $makeFlags "''${makeFlagsArray[@]}" \
-            $installFlags "''${installFlagsArray[@]}"
-          unlink $out/lib/modules/${modDirVersion}/build
-          rm -f $out/lib/modules/${modDirVersion}/source
+        postInstall =
+          ''
+            mkdir -p $dev
+            cp vmlinux $dev/vmlinux
```

```suggestion
            cp vmlinux $dev/
```

Saves one unnecessary change and rebuild on linux when `isModular`